package.json

```diff
@@ -77,5 +77,8 @@
         "tslint": "5.12.1",
         "typescript": "4.4.3",
         "typescript-standard": "^0.3.36"
+    },
+    "dependencies": {
+        "cbor": "^8.1.0"
```

Comment:
I accidentally added this dependency to the wrong package – I will update the PR shortly.